app/puppet-code/puppet-code.go

```diff
@@ -0,0 +1,37 @@
+package app
+
+import (
+	"github.com/puppetlabs/pe-sdk-go/app/puppet-code/api"
+	"github.com/puppetlabs/pe-sdk-go/token"
+	"github.com/puppetlabs/pe-sdk-go/token/filetoken"
+)
+
+// PuppetCode FIXME
+type PuppetCode struct {
+	ServiceURL string
+	Cacert     string
+	TokenFile  string
+
+	Client api.Client
+	Token  token.Token
```

Comment:
We should have token only as string in app, all token-file related code should used only from cli